### PR TITLE
added packages/molenc/molenc.0.0.1

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/molenc/molenc.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/molenc"
+bug-reports: "https://github.com/UnixJunkie/molenc/issues"
+dev-repo: "git+https://github.com/UnixJunkie/molenc.git"
+license: "BSD-3"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+remove: ["sh" "-c" "rm -f %{bin}%/molenc_type_atoms.py"]
```
Comment (kit-ty-kate) [inline packages/molenc/molenc.0.0.1/opam]:
```suggestion
remove: ["rm" "-f" "%{bin}%/molenc_type_atoms.py"]
```
I don't see any need for `sh` here

Comment (kit-ty-kate) [reply packages/molenc/molenc.0.0.1/opam]:
```suggestion
```
Alternatively the `remove` field can also be entirely discarded as opam 2 does track the files installed by the package.

**Comment by UnixJunkie** [comment]
ok, I removed the remove field.

**Comment by UnixJunkie** [comment]
CI tests fail on the Mac.
Probably because conf-rdkit should be updated so that it installs in there.

**Comment by mseri** [comment]
This seems good to merge

**Pull request merged**